.env

```diff
@@ -1,4 +1,4 @@
-REACT_APP_API_URL=https://1yx9acvrod.execute-api.us-east-1.amazonaws.com/dev
+REACT_APP_API_URL=https://c254cnt0kd.execute-api.us-east-1.amazonaws.com/dev
```

Comment:
this is just for development time right?

hmm, is there a way we can make this more consistent? probably not easily or for free (thinking you'd need Route53 or API gateway involved to give you a consistent url.

Reply:
that's the strange thing, the dev endpoint has always been the same except today when I deployed latest changes via my command line